python/modules/IcePy/Util.h

```diff
@@ -89,7 +89,7 @@ namespace IcePy
         //
         // Convert the Python exception to its C++ equivalent.
         //
-        void raise();
+        void raise(bool includeStackTrace);
```

Comment:
From reading the code, that's not really what raise() does. It converts the Python exception into one of the 6 special Ice local exceptions. And as a result, we don't need an includeStackTrace parameter for this function.